### DISCUSSION TITLE
Can i know the menu placement open when i set menuPlacement="auto"

### DISCUSSION THREAD

**Discussion by zaherkassem** [JedWatson/react-select#5462] [Q&A]
is there any props that tell me where is the menuPlacement top or bottom. ? can we add a className on the top dev like menuPlacement-top or menuPlacement-bottom for each option. ?

**Comment by lukethacoder** [comment]
Would love to know if there is a way to check if the select has been rendered `top` or `bottom` dynamically when setting `menuPlacement="auto"`.

Looks like https://github.com/JedWatson/react-select/discussions/4872 somewhat covers this, but doesn't handle the `"auto"` option.
